mkdocs.yml

```diff
@@ -0,0 +1,58 @@
+site_name: Workflows
+repo_url: https://github.com/scality/workflows
+repo_name: scality/workflows
+edit_uri: edit/main/docs/
+theme:
+  icon:
+    repo: fontawesome/brands/github
+  name: material
+  include_search_page: false
+  search_index_only: true
+  language: en
+  features:
+    - navigation.indexes
+    - navigation.instant
+    - navigation.sections
+    - navigation.expand
+    - navigation.tabs
+    - navigation.tabs.sticky
+    - search.highlight
+    - search.share
+    - search.suggest
+    - toc.follow
+    - header.autohide
+  palette:
+    # Palette toggle for light mode
+    - scheme: default
+      toggle:
+        icon: material/brightness-7
+        name: Switch to dark mode
+
+    # Palette toggle for dark mode
+    - scheme: slate
+      toggle:
+        icon: material/brightness-4
+        name: Switch to light mode
+  font:
+    text: Roboto
+    code: Roboto Mono
+
+nav:
+- Docker build: docker-build.md
+- Known issues: known-issues.md
```

Comment:
Would be good to keep a quick intro to the site, nothing crazy, so add a `docs/index.md` file with someting like:

```markdown
# Workflows

This repository is used to share GitHub Actions reusable workflows across the organization.

```
You can add link to the original doc of what is a reusable workflow, and that's it. 

```suggestion
nav:
- Home: index.md
- Docker build: docker-build.md
- Known issues: known-issues.md
```